### PR TITLE
Renaming Classes

### PR DESCRIPTION
**Pull request by boldCM**
renamed almost every class using Bem- would love some Feedback: if the naming makes sense to you, if it is reusable in a good way, etc.

### CODE REVIEW

**Review by mhstrkmp — COMMENTED** [review]
lgtm 🎉

File: src/components/cardPlaylist/cardPlaylist.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
   <div class="cardPlaylist">
     <h5>My Top Tracks</h5>
     <br />
-    <p class="Description">100 Songs</p>
+    <p class="cardPlalist__description">100 Songs</p>
```
Comment (kschmitz123) [inline src/components/cardPlaylist/cardPlaylist.html:11]: Typo in playlist

**Review by kschmitz123 — APPROVED** [review]
Looks great:)

File: src/components/cardPlaylist/cardPlaylist.html (hunk 1 above)
Comment (boldCM) [reply src/components/cardPlaylist/cardPlaylist.html:11]: ahh, thank you!:)

File: src/components/list/list.css [hunk 2]
```diff
@@ -25,28 +25,30 @@
   border: none;
 }
 
-.btn-sidebar > img {
+.sidebar__btn > img {
   margin-top: 5px;
   margin-bottom: 5px;
 }
-.iconAudio {
+.sidebar__iconAudio {
```
Comment (AlexisRoe) [inline src/components/list/list.css:32]: mabye better with .sidebar__icon--audio
because its another version of icon?

Comment (boldCM) [reply src/components/list/list.css:32]: yes!!! good point! 🤩
thank you :)

**Pull request merged**